packages/terra-framework-docs/src/terra-dev-site/doc/compact-interactive-list/Examples.2/ResponsiveColumns.2.doc.mdx

```diff
@@ -0,0 +1,51 @@
+import { Notice } from '@cerner/terra-docs';
+import ResponsiveColumns from './ResponsiveColumns?dev-site-examples';
+import ResponsiveColumnsMaxWidth from './ResponsiveColumnsMaxWidth?dev-site-examples';
+
+# Responsive Columns
+
+### Description
+
+This example demonstrates a [Compact Interactive List](/components/cerner-terra-framework-docs/compact-interactive-list/about) with responsive columns.
+
+* The middle semantic column with the text information has [_flexGrow_](/components/cerner-terra-framework-docs/compact-interactive-list/about#column) prop set to `true` which allows it to grow to fill up the container's width.
+* The maximum width for the text column is not limited, as the [_maximumWidth_](/components/cerner-terra-framework-docs/compact-interactive-list/about#column) property is not set.
+* The minimum width would take the default value as [_minimumWidth_](/components/cerner-terra-framework-docs/compact-interactive-list/about#column) property is not set.
+* Once the list reaches its minimum width, defined by fixed width columns and the default minimum width of the flex growing column, the scroll shows up.
+
+<ResponsiveColumns />
+
+### Usage
+
+#### Creating a Responsive Column
+
+There are two ways to make a sematic column responsive:
```

Comment:
Nit: typo
```suggestion
There are two ways to make a semantic column responsive:
```

Reply:
Great suggestion, and much appreciated, but it looks like all wording and styling comment were outdated per this commit by @cldeitrich: https://github.com/cerner/terra-framework/pull/1950/commits/62c72c0098e6edb05ec923b22cf997650ac247bf